prisma/schema.prisma

```diff
@@ -8,10 +8,78 @@ datasource db {
   directUrl = env("DATABASE_URL_DIRECT")
 }
 
+model Event {
+  id  String  @id
+  creator  String //User id (from clerk)
+  creation_date  DateTime  @default(now())
```

Comment:
I think it'd be good if we had an updatedAt as well. What do you think about normalising all timestamp-related fields so they'd be in the format `createdAt`, `updatedAt`, `deletedAt`? Or maybe `createdDate`, `updatedDate`, etc...